client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -319,6 +319,11 @@ module.exports = {
 	},
 
 	domainManagementTransferToOtherUser( pageContext ) {
+		const selectedSite = sites.getSelectedSite();
+		if ( selectedSite && selectedSite.options.is_automated_transfer ) {
```

Comment:
I'm not sure, but is there a case where we might have `selectedSite` but not `selectedSite.options`? If so this could fatal.